build.gradle

```diff
@@ -321,6 +321,8 @@ project('segmentstore:storage:impl') {
         compile group: 'org.rocksdb', name: 'rocksdbjni', version: rocksdbjniVersion
         testCompile project(':test:testcommon')
         testCompile project(path:':segmentstore:storage', configuration:'testRuntime')
+        testCompile group: 'io.netty', name: 'netty-transport-native-epoll', version: nettyVersion, classifier: 'linux-x86_64'
```

Comment:
Why do we need to add these test compile dependencies?

Reply:
I thought it was a problem of not having 'epoll' available, I can drop this line if it is not needed (once I see all tests passing)

Reply:
it is not needed. dropped